tests/bluetooth/tester/src/btp_csip.c

```diff
@@ -192,12 +192,38 @@ static uint8_t btp_csip_discover(const void *cmd, uint16_t cmd_len,
 static uint8_t btp_csip_set_coordinator_lock(const void *cmd, uint16_t cmd_len, void *rsp,
 					     uint16_t *rsp_len)
 {
+	const struct bt_csip_set_coordinator_set_member *members[ARRAY_SIZE(btp_csip_set_members)];
 	const struct btp_csip_set_coordinator_lock_cmd *cp = cmd;
+	unsigned long member_count = 0;
 	int err;
 
 	LOG_DBG("");
 
-	err = bt_csip_set_coordinator_lock(btp_csip_set_members, cp->count, &cur_csis_inst->info);
+	if (cp->addr_cnt != 0) {
+		/* TODO: add support for lock request procedure on subset of set members */
+		/* If no address is passed, we perform procedure on all members */
+		return BTP_STATUS_FAILED;
+	}
```

Comment:
> /* TODO: add support for lock request procedure on subset of set members */

Isn't that was this PR was supposed to add?

> /* If no address is passed, we perform procedure on all members */

Is this another TODO? The comment doesn't match the implementation

Reply:
Right, this comment and commit msg don't match with PR content. Fixed it